src/node/coinstats.cpp

```diff
@@ -51,6 +51,7 @@ bool GetUTXOStats(CCoinsView *view, CCoinsStats &stats)
     }
     ss << stats.hashBlock;
     uint256 prevkey;
+    unsigned int coins_count{0};
```

Comment:
I don't really like that `coins_count` is initialized to zero at the beginning of the function, but the other members of `stats` aren't. I'd prefer if all stats are initialized in one go: 

```cpp
stats = CCoinsStats{}; // reset stats

Reply:
@jamesob ping